### ISSUE TITLE
Issues with Apps on GrapheneOS Pixel 3a xl

### ISSUE THREAD

**Issue by koolandrew** [GrapheneOS/os-issue-tracker#1789]
The phone was working fine with very light use, and all of a sudden, many of the apps, have difficulty opening, or will open and stay open for about one second. I have downloaded very few from the play store, and the play store itself was having issues. Most of the apps are from F droid or Aurora. I dont seem to have any issues with the apps from F droid, mostly from Aurora and Apkpure.

It is interesting that the play store seems to be forcing google play services for AR, but that isnt happening on my Samsung running regular Android. I am not sure if it is related.

I am not sure what else i can do but wipe it and start over, but i thought i would report this in case you wanted any further investigation.

**Comment by thestinger** [comment]
It sounds like you updated Play services via Play Store to a newer unsupported version. Since we aren't improving the sandboxed Google Play compatibility layer for legacy 3rd generation devices anymore, that will cause major breakage.

You likely just need to uninstall Play services and the Play Store followed by reinstalling them from our app repository client. Reject updates to Play services and Play Store via the Play Store going forward.

**Comment by thestinger** [comment]
Also, don't remove and reinstall GSF, just Play services and Play Store. Removing/reinstalling GSF will break apps for another reason due to an AOSP / stock OS bug where apps aren't given auto-granted permissions defined by non-OS apps when the app using them is installed before the app declaring them. We fixed this for GrapheneOS downstream already but 3rd gen Pixels aren't getting most GrapheneOS changes since August when Android 13 was released and we moved to it.
